src/channeldef.ts

```diff
@@ -280,7 +280,7 @@ export interface TypeMixins<T extends Type> {
    * The type of measurement (`"quantitative"`, `"temporal"`, `"ordinal"`, or `"nominal"`) for the encoded field or constant value (`datum`).
    * It can also be a `"geojson"` type for encoding ['geoshape'](https://vega.github.io/vega-lite/docs/geoshape.html).
    *
-   * Since Vega-Lite 4.14.0, Vega-Lite automatically infers data types in many cases as discussed below. However, type is required for a field if:
```

Comment:
Why removing this clause?

Reply:
We don't mention specific versions in other places, do we?